rclcpp_components/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ find_package(ament_cmake_ros REQUIRED)
 find_package(ament_index_cpp REQUIRED)
 find_package(class_loader REQUIRED)
 find_package(composition_interfaces REQUIRED)
+find_package(domain_bridge REQUIRED)
```

Comment:
Do we really want to make `domain_bridge` a dependency of `rclcpp_components` like this?  It means bringing `domain_bridge` into the ROS 2 core, so we'd have to add it to the package.xml here, and also to https://github.com/ros2/ros2/blob/master/ros2.repos .  But we should seriously consider if that is what we really want to do; it means that `domain_bridge` becomes release-critical (i.e. we cannot release without it).

I'd much rather see us try to do something like this inside of the `domain_bridge` package instead, and keep it out of the ROS 2 core.

Reply:
Came here to ask this too. I don't think we want it as a dependency of `rclcpp_components`.

> I'd much rather see us try to do something like this inside of the domain_bridge package instead, and keep it out of the ROS 2 core.

👍

Reply:
+1 I think this executable should live in the domain_bridge package.

Reply:
Makes sense, thanks for your comments. I've moved `domain_bridge_container.cpp` into `domain_bridge`, removing the need for the dependency.